go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -227,9 +227,20 @@ func runOvnKube(ctx *cli.Context) error {
 			return err
 		}
 		watchFactory = masterWatchFactory
-		var ovnNBClient, ovnSBClient goovn.Client
+		var ovnSBClient goovn.Client
+		var ovnNBClient goovn.Client
+		ovnNBClients := make([]*ovn.OVNNBClient, 10)
 		var libovsdbOvnNBClient, libovsdbOvnSBClient libovsdbclient.Client
 
+		// create 10 NB Clients
+		for i := range ovnNBClients {
```

Comment:
this is going to consume 10x more memory? Each client will subscribe to all the tables to cache.

Reply:
yeah, idea was to just see if this helps in the scale tests, if it does then we can look at a utility to create multiple clients that share the same cache in go-ovn lib or libovsdb perhaps. Also we can quantify how big the cache is in the scale lab.